BankApplicationPractice/BankLibrary/Bank.cs

```diff
@@ -49,30 +51,53 @@ public void HandlerNextDay()
             }
         }
 
-        public void OpenCell(int data, string key, Action<string> handler)
+        public int AddLocker(string keyword, object data)
         {
-            var newCell = new BankCell<int>()
-            {
-                Id = _bankCell.Count,
-                Data = data,
-                _handlerCell = handler
-            };
+            var keyLocker = new KeyLocker(_lockers.Count + 1, keyword);
+
+            _lockers.Add(keyLocker, data);
+
+            return keyLocker.Id;
+        }
 
-            newCell.Open();
+        public object GetLockerData(int id, string keyword)
+        {
+            foreach (var locker in _lockers)
```

Comment:
use key to find the value, instead of enumerating the whole dictionionary
`return _lockers[key]`